README.EN.md

```diff
@@ -64,15 +65,15 @@ List of robotics initiatives in Brazil! Feel free to [contribute](CONTRIBUTING.m
 - [GRACO - Grupo de Automação e Controle (UnB)](http://www.graco.unb.br/)
 - [IDR Lab - Laboratório de Robótica Industrial e de Defesa (IME)](http://labmec.imejunior.com.br/)
 - [LABSCA - Laboratório de Sistemas Computacionais Autônomos (ITA)](http://www.comp.ita.br/labsca/)
-- [LaR - Laboratório de Robótica (UFBA)](http://www.dee.eng.ufba.br/lar/)
+- [LaR](http://www.dee.eng.ufba.br/lar/) - Laboratório de Robótica (UFBA) [:octocat:](https://github.com/lar-deeufba)
```

Comment:
I think it is better to keep it like this
```suggestion
- [LaR - Laboratório de Robótica (UFBA) ](http://www.dee.eng.ufba.br/lar/) - [:octocat:](https://github.com/lar-deeufba)
```
Placing the symbol after the dash shall not raise errors.